src/_P109_ThermOLED.ino

```diff
@@ -171,13 +171,15 @@ boolean Plugin_109(uint8_t function, struct EventStruct *event, String& string)
       }
 
       addFormPinSelect(PinSelectPurpose::Generic_input,  F("Button left/down"), F("taskdevicepin1"), CONFIG_PIN1);
+      addFormCheckBox(F("Button left/down Pull mode (0=IntPullUp, 1=Input)"), F("invertbutton1"), P109_GET_BUTTON1_INVERT);
```

Comment:
Hmm this is quite confusing.
Unchecked is pulled up?

What do you want to tell the user? Is it about the pull-up or invert up/down button?

Reply:
Renamed variables.